src/components/eip3643/Admin.tsx

```diff
@@ -30,7 +30,7 @@ export default function Admin() {
           readTokenName({}, tokenAddress),
           readTokenOwner({}, tokenAddress),
         ]).then((res) => {
-          res[1][0].toString() === accountId?.toString() &&
+          res[1][0].toString().toLowerCase() === accountEvm?.toLowerCase() &&
```

Comment:
we potentially need more safe checks to avoid "of undefined" for any parts missing of `res[1][0].toString().toLowerCase()`